.packit.yaml

```
@@ -0,0 +1,28 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: criu.spec
+
+# add or remove files that should be synced
+synced_files:
+    - criu.spec
+    - .packit.yaml
+    - criu-tmpfiles.conf
+
+current_version_command: ["make", "show-version"]
```

Regarding the failures we discussed yesterday: deprecating the `current_version_command` is in our [backlog](https://github.com/packit/packit/issues/1075), please try using the `get-current-version` action instead. I've run a few experiments and it seems this should fix your issue.

```suggestion
actions:
  get-current-version:
    - "make show-version"
```